src/platforms/ctxlink/Makefile.inc

```diff
@@ -0,0 +1,47 @@
+CROSS_COMPILE ?= arm-none-eabi-
+BMP_BOOTLOADER ?=
+CC = $(CROSS_COMPILE)gcc
+OBJCOPY = $(CROSS_COMPILE)objcopy
+
+CFLAGS += -Istm32/include -mcpu=cortex-m4 -mthumb \
+	-mfloat-abi=hard -mfpu=fpv4-sp-d16 \
+	-DSTM32F4 -DCTX_LINK -I../deps/libopencm3/include \
+	-Iplatforms/common/stm32
+
+LDFLAGS_BOOT = -lopencm3_stm32f4 \
+	-Tctxlink.ld -nostartfiles -lc -lnosys \
+	-Lplatforms/ctxlink \
+	-Wl,-Map=mapfile -mthumb -mcpu=cortex-m4 -Wl,-gc-sections \
+	-mfloat-abi=hard -mfpu=fpv4-sp-d16 \
+	-L../deps/libopencm3/lib
+
+ifeq ($(BMP_BOOTLOADER), 1)
+$(info  Load address 0x08004000 for BMPBootloader)
+LDFLAGS = $(LDFLAGS_BOOT) -Wl,-Ttext=0x8004000
+CFLAGS += -DDFU_SERIAL_LENGTH=9
+else
+LDFLAGS += $(LDFLAGS_BOOT)
```

Comment:
`LDFLAGS` should not be defined already in this branch (and if it is, wants replacing) so we suggest this be switched from `+=` to just `=` same as the first branch.